BrightCoders-Institute/s10a1-proyecto-final-team-01-s10a1-proyecto-final
# Internal Application's Blog

# Description

- Added a blog that is exclusive for the application's users.
- Added posting system that is available for any user.
- Added posts commenting feature.

## Testing

- Run the rails db:migrate to properly create the database in a local PostgreSQL environment.
- Install Action Text and Active Storage.
- Run the rails db:seed command in order to load all the default users data.
- Run either bin/dev or rails server and open the localhost:3000 endpoint in any browser.
- Login with any account in order to check the blog.
